extensions-contrib/opencensus-extensions/src/main/java/org/apache/druid/data/input/opencensus/protobuf/OpenCensusProtobufInputRowParser.java

```diff
@@ -88,22 +107,30 @@ public OpenCensusProtobufInputRowParser withParseSpec(ParseSpec parseSpec)
       throw new ParseException(e, "Protobuf message could not be parsed");
     }
 
+    // Process metric descriptor labels map keys.
+    List<String> descriptorLabels = metric.getMetricDescriptor().getLabelKeysList().stream()
+        .map(s -> this.metricLabelPrefix + s.getKey())
+        .collect(Collectors.toList());
+
+    // Process resource labels map.
+    Map<String, Object> resourceLabelsMap = new HashMap<>(metric.getResource().getLabelsMap().entrySet().stream()
```

Comment:
is the `new HashMap<>()` necessary here? doesn't the `Collectors.toMap` already create the map?

Reply:
Removed, seems like I am missing important details. Will be more stringent in future, apologies.